### ISSUE TITLE
Additional values for idno/@type

### ISSUE THREAD

**Issue by xlhrld** [deutschestextarchiv/dtabf#43]
In one of the projects I'm affiliated with we base the relation among lexicographical citations and the cited text on PPN. Thus, we need to record PPNs in the texts' metadata – the natural place being idno. So idno/@type="PPN" would come in handy. While I'm at it: one can also imagine use-cases for other bibliographic IDs such as OCLC.

I'm aware of the benefits in keeping the list restricted but I would opt for its extension with at least "PPN".

**Comment by textloop** [comment]
I agree - the list of values should be closed, but should be enriched with all values, that are referring to widly spread standards (like PPN),

**Comment by susannehaaf** [comment]
Ok from my side.

**Comment by funkyfuture** [comment]
is it at all useful to constraint the values outside a concrete project? the current constraints are good recommendations though.

**Comment by xlhrld** [comment]
@funkyfuture: It is useful because those constraints allow for interoperability. The fixed semantics of the values for `idno/@type` – by way of defining them in the DTABf – allow us to be sure we refer to the same system. This is much less clear when we allow `idno/@type="my-fancy-reference-system"`.

**Comment by xlhrld** [comment]
Sorry, wrong button …

**Comment by burki** [comment]
Adding PPN seems fine, I'd suggest to add DOI as well (as an alternative to URN)

**Comment by funkyfuture** [comment]
@xlhrld but what exactly are the requirements for interoperability? the structure how the information is to be encoded is exactly defined, and thus allows implementers to "[en]sure we refer to the same system". whether the software is capable of dealing with all provided information or not doesn't directly relate to its operability.

the set of allowed values is already quiet long and i don't see that all implementations of software to process DTABf encoded data are interested in all of the information (cabinet archives are a great example, and what is TCF anyway?), they will just ignore these. and that is a sane behaviour imo.

to make that clear (as with regards to the `type` attribute in other contexts): a recommended, unambiguously defined vocabulary is great to avoid reinventions.

given the age of this issue, the dogma also doesn't seem to align well with the resources that are allocated to the standartisation process.

**Comment by xlhrld** [comment]
@funkyfuture Wrt. `idno/@type` and `idno` in general, the requirements for interoperability in this context are: a) an unambiguously specified reference system (URN, PPN, what have you) and b) IDs compliant to the specified reference system. Unambiguous specification of the reference system is ensured by the fixed set of (documented) values for `idno/@type`. (My point in opening this ticket was to provide just another widely used reference system.)

If a given *implementation* doesn't know how to deal with a certain reference scheme it should ignore it, true. If an *implementer* doesn't know which reference scheme is denoted by the value of `URN` she should find a description of what exactly is meant by it in the DTABf schema and documentation. She should _not_ have to guess what `URN` actually means in the context of the DTABf. She should also _not_ infer the reference system from the structure of the information. (This is how I interpret your second sentence.) Given the value `B4 F 37-1` you cannot reliably infer the underlying reference system. There may well be even several systems in which `B4 F 37-1` is a valid reference.

Please clarify if I got you wrong.

**Comment by xlhrld** [comment]
@susannehaaf: at http://deutschestextarchiv.de/doku/basisformat/mdSdMsDesc.html of all the possible values for `idno/@type` only for are actually specified (`shelfmark`, `URLCatalogue`, `URLImages`, `PPN`). The other values (such as `URLTCF`) should also be documented.

**Comment by susannehaaf** [comment]
I pushed your last comment into a new ticket, @xlhrld .
Closing this ticket as fixed (version 1.1.0).

**Comment by tboenig** [comment]
I think we must reopen the issue.
How do we handle **ISBN, ISSN**.
I recall the comment: https://github.com/deutschestextarchiv/dtabf/issues/43#issuecomment-420905853.
My suggestion is we include these identifiers.

**Comment by tboenig** [comment]
Excuse me. I've revisited the subject.
An ISBN, ISSN is very well coded.
ISBN, ISSN can be coded as URN. 
```xml
<idno type="URN">urn:isbn:978-3-8362-6537-9</idno>
```
Please complete this in the documentation.

**Comment by tboenig** [comment]
Hi, that's the value of attribute "type" must be equal to "DOI", "DTADirName", "DTAID", "EPN", "PIDCMDI", "PPN", "URLCAB", "URLCatalogue", "URLHTML", "URLIIIF", "URLImages", "URLTCF", "URLText", "URLWeb", "URLXML", "URN", "VD" or "shelfmark"
I miss the OCR-formats like ALTO, PAGE-> URLALTO, URLPAGE. Is this possible?

**Comment by StefanDumont** [comment]
SG vom 6.2.2026:
* Überprüfen ob alles im Schema auch dokumentiert ist.
* Schema und Dokumentation ergänzen um: 
  * Handle
  * DOI (falls es noch fehlt)
  * ISSN
  * ISBN
  * ALTO (? - @tboenig überlegt nochmal)
  * PAGE (? - @tboenig  überlegt nochmal)

**Comment by StefanDumont** [comment]
@tboenig Schaust du nochmal auf den vorangehenden Kommentar zu ALTO und PAGE und gibst hier kurz Feedback, ob diese Werte Sinn machen würden? Danke!
